packages/compass-web/src/entrypoint.tsx

```diff
@@ -276,6 +277,16 @@ const CompassWeb = ({
   );
   const autoConnectConnectionId = initialWorkspaceRef.current?.connectionId;
 
+  const onTrackRef = useRef(onTrack);
+
+  const telemetryOptions = useRef<TelemetryServiceOptions>({
+    sendTrack: (event: string, properties: Record<string, any> | undefined) => {
+      onTrackRef.current && void onTrackRef.current(event, properties || {});
+    },
+    logger,
+    preferences: preferencesAccess.current,
```

Comment:
I think there is an unexpected side-effect due to finally correctly passing all required preferences to compass web tracking service: `trackUsageStatistics` defaults to `false` so tracking is disabled, you'd need to set it to true in the `ReadOnlyPreferenceAccess` instance created a few lines above this one so that the tracking events are actually emitted (should also fix the failing compass-web tests)

Reply:
asking if I should create a ticket to get the actual value

Reply:
and thanks for noticing!

Reply:
It's the actual behavior, from compass-web perspective the telemetry is always "on", whether or not mms will send it will be derived with their internal logic